frontend/src/metabase/query_builder/components/notebook/NotebookDataPicker/NotebookDataPicker.tsx

```diff
@@ -57,6 +58,30 @@ export function NotebookDataPicker({
     [query, stageIndex, table],
   );
 
+  const pickerInfo = useMemo(
```

Comment:
nit: `pickerInfo` is not used in render. We can compute it in `getUrl` only

Reply:
Not a nit.
Great call!

Thank you